compute_endpoint/packaging/JenkinsFile

```diff
@@ -20,11 +20,22 @@ pipeline {
             agent {label "package_creator"}
             steps {
                 script {
+
+                    // scmVars.GIT_BRANCH comes from the Jenkins UI at Pipeline -> Definition -> SCM
+                    //   https://builds.globus.org/jenkins/job/BuildGlobusComputeAgentPackages/configure
                     def scmVars = checkout scm
-                    env.BRANCH_NAME = scmVars.GIT_BRANCH.replaceFirst(/^.*origin\//, "")
+                    env.DEFAULT_BRANCH = scmVars.GIT_BRANCH.replaceFirst(/^.*origin\//, "")
                     env.SOURCE_STASH_NAME = "${UUID.randomUUID()}"
-                    echo "env.BRANCH_NAME = ${env.BRANCH_NAME}"
+                    echo "Using env.DEFAULT_BRANCH = ${env.BRANCH_NAME}"
+
+                    // BRANCH_OR_TAG and BUILD_FOR_STABLE comes from the Jenkins UI string and boolean
+                    //   parameters which are the two options presented on the main Compute build page
                     sh "git clean -fdx"
+                    env.BRANCH_NAME = params.BRANCH_OR_TAG
+                    env.BUILD_FOR_STABLE = params.BUILD_FOR_STABLE
+                    env.TAG_NAME = "${env.BRANCH_NAME}"
```

Comment:
I've forgotten the later-on Groovy script innards; would it behoove us to set one OR the other?

Meanwhile, if setting both (in direct answer to my question :point_up:), perhaps clearer to colocate `BRANCH_NAME` and `TAG_NAME` since they're to be the same value.  Consider:
```groovy
env.BRANCH_NAME = params.BRANCH_OR_TAG
env.TAG_NAME = params.BRANCH_OR_TAG
env.BUILD_FOR STABLE = params.BUILD_FOR_STABLE
```

Reply:
Same reason - I wasn't sure if TAG_NAME is used later, so left it set just in case.  I made a bit more clearer as you suggested but I'll also look to see if TAG_NAME matters.